bin.ts

```diff
@@ -52,85 +52,90 @@ const PRESELECTED_PROJECT_OPTIONS = {
-    default: false,
-    describe: "Don't send telemetry data to Sentry",
-    type: 'boolean',
-  },
-  'promo-code': {
-    alias: 'promo-code',
-    describe: 'A promo code that will be applied during signup',
-    type: 'string',
-  },
-  'force-install': {
-    default: false,
-    describe: 'Force install the SDK NPM package',
-    type: 'boolean',
-  },
-  ...PRESELECTED_PROJECT_OPTIONS,
-}).argv;
+const argv = yargs(hideBin(process.argv), process.cwd())
+  .options({
+    debug: {
+      default: false,
+      describe: 'Enable verbose logging\nenv: SENTRY_WIZARD_DEBUG',
+      type: 'boolean',
+    },
+    uninstall: {
+      default: false,
+      describe: 'Revert project setup process\nenv: SENTRY_WIZARD_UNINSTALL',
+      type: 'boolean',
+    },
+    'skip-connect': {
+      default: false,
+      describe:
+        'Skips the connection to the server\nenv: SENTRY_WIZARD_SKIP_CONNECT',
+      type: 'boolean',
+    },
+    quiet: {
+      default: false,
+      describe:
+        'Do not fallback to prompting user asking questions\nenv: SENTRY_WIZARD_QUIET',
+      type: 'boolean',
+    },
+    i: {
+      alias: 'integration',
+      choices: Object.keys(Integration),
+      describe:
+        'Choose the integration to setup\nenv: SENTRY_WIZARD_INTEGRATION',
+    },
+    p: {
+      alias: 'platform',
+      choices: Object.keys(Platform),
+      describe: 'Choose platform(s)\nenv: SENTRY_WIZARD_PLATFORM',
+      type: 'array',
+    },
+    u: {
+      alias: 'url',
+      describe: 'The url to your Sentry installation\nenv: SENTRY_WIZARD_URL',
+    },
+    project: {
+      type: 'string',
+      describe: 'The Sentry project slug to use',
+      defaultDescription: 'Select project during setup',
+      default: undefined,
+    },
+    org: {
+      type: 'string',
+      describe: 'The Sentry org slug to use',
+      defaultDescription: 'Select org during setup',
+      default: undefined,
+    },
+    saas: {
+      default: false,
+      describe: 'Skip the self-hosted or SaaS URL selection process',
+      defaultDescription: 'Select self-hosted or SaaS during setup',
+      type: 'boolean',
+    },
+    s: {
+      alias: 'signup',
+      default: false,
+      describe: 'Redirect to signup page if not logged in',
+      type: 'boolean',
+    },
+    'disable-telemetry': {
+      default: false,
+      describe: "Don't send telemetry data to Sentry",
+      type: 'boolean',
+    },
+    'promo-code': {
+      alias: 'promo-code',
+      describe: 'A promo code that will be applied during signup',
+      type: 'string',
+    },
+    'force-install': {
+      default: false,
+      describe: 'Force install the SDK NPM package',
+      type: 'boolean',
+    },
+    ...PRESELECTED_PROJECT_OPTIONS,
+  })
+  // This prevents `yargs` from trying to read the local package.json
+  // as it's not available in the Node Single Executable Binary artifacts versions
+  .version(process.env.npm_package_version as string).argv;
```

Comment:
Is this env variable always set? If so TIL 😅

Reply:
It is always when using `fossilize` or `npx`. Otherwise it may not be set.

Reply:
ok, since we fall back to other methods that's fine, thx!